src/sphinx_tags/__init__.py

```diff
@@ -172,10 +172,12 @@ def create_file(
         """
         # Get sorted file paths for tag pages, relative to /docs/_tags
         tag_page_paths = sorted([i.relpath(srcdir) for i in items])
+        ref_label = f"sphx_tag_{self.file_basename}"
```

Comment:
Do we want to use `file_basename` or `name` ? Since `file_basename` will be normalized (with no emojis or special characters), would this be better for ref label? (I'm not sure)

see: https://github.com/melissawm/sphinx-tags/pull/86#discussion_r1421798846

Reply:
Right, `file_basename` is the better option here, since `name` could potentially be invalid as a ref label.

Valid characters for `ref` are listed here: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#reference-names

Reply:
Whoops I somehow thought you used the other one. Nevermind!

Reply:
No problem! Thanks for taking a look.